homeassistant/components/asuswrt/sensor.py

```diff
@@ -40,21 +40,24 @@ class AsusWrtSensorEntityDescription(SensorEntityDescription):
     """A class that describes AsusWrt sensor entities."""
 
     factor: int | None = None
+    unique_identifier: str | None = None
 
 
 UNIT_DEVICES = "Devices"
 
 CONNECTION_SENSORS: tuple[AsusWrtSensorEntityDescription, ...] = (
     AsusWrtSensorEntityDescription(
         key=SENSORS_CONNECTED_DEVICE[0],
-        name="Devices Connected",
+        unique_identifier="Devices Connected",
```

Comment:
Instead of keeping this behavior in, we should first refactor these out and migrate it to a sane identifier.

Reply:
Downside is that they are used for the unique id. I've seen some examples of migrating unique_id so I'll check them out and apply it here